packages/pharos/src/components/tabs/pharos-tabs.test.ts

```diff
@@ -244,4 +244,27 @@ describe('pharos-tabs', () => {
     await component.updateComplete;
     expect(count).to.equal(0);
   });
+
+  it('does not render panel separator', async () => {
+    const spanElement = component.renderRoot.querySelector('.panel-separator') as HTMLDivElement;
+
+    expect(spanElement).to.be.null;
+  });
+
+  it('renders panel separator', async () => {
+    component = await fixture(html`
+      <pharos-tabs panelseparator="true">
```

Comment:
```suggestion
      <pharos-tabs panel-separator>
```
For boolean props, the presence of its attribute is truthy

Reply:
Is this preferred? I added `true` so that it was verbose.

Reply:
Strictly speaking, `true` is not an allowed value—the value must be one of:

* unassigned
* empty string
* a case-sensitive match for the attribute name itself

See more on the MDN docs [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#boolean_attributes).

Reply:
Yes since otherwise it may be misleading for folks and they might think they could do `panel-separator="false"` which is not the case.

Reply:
Wow, boolean attributes have different rules than I assumed. Thanks for that info!